animations/spells/1st/ignition.json

```diff
@@ -9,7 +9,6 @@
 					"file": "jb2a.cast_generic.fire.side01.orange.0",
 					"options": {
 						"waitUntilFinished": -1200,
-						"rotateTowards": {},
```

Comment:
I will have to check but this should most likely be substituted for "true" basically.